heron/api/src/java/BUILD

```diff
@@ -20,13 +20,22 @@ api_deps_files =  \
         "//heron/common/src/java:basics-java",
     ]
 
+# Legacy Low Level Api
```

Comment:
why "Legacy"? My understanding is that this API should be suggested in some cases. Same for the functional comment. It is not "External".

Reply:
fixed